### ISSUE TITLE
Docs search is broken

### ISSUE THREAD

**Issue by ewels** [nextflow-io/nextflow#2040]
Since a week or more ago, the docs search just hangs and never loads: https://www.nextflow.io/docs/latest/search.html?q=duration&check_keywords=yes&area=default

The browser console shows this:

```console
Uncaught ReferenceError: Stemmer is not defined
    at Object.query (searchtools.js:158)
    at Object.setIndex (searchtools.js:98)
    at <anonymous>:1:8
    at b (jquery.js:2)
    at Function.globalEval (jquery.js:2)
    at text script (jquery.js:2)
    at jquery.js:2
    at l (jquery.js:2)
    at XMLHttpRequest.<anonymous> (jquery.js:2)
```

**Comment by pditommaso** [comment]
@abhi18av can you give it a look ?

**Comment by abhi18av** [comment]
Sure, I gave it a look and here's what I found.

1. The problem is due to a bug in the newer `sphinx` release as mentioned here https://github.com/sphinx-doc/sphinx/issues/8623 and https://github.com/neuronsimulator/nrn/issues/888.
2. I think we can mitigate it by pinning the `sphinx` version on the CI
3. I tried building locally with the `sphinx-2.1.2` and the search is working fine.

**Comment by pditommaso** [comment]
Solved upgrading sphynx theme to 0.5.2 4a165f43

**Comment by LyleLee** [comment]
> Solved upgrading sphynx theme to 0.5.2 [4a165f4](https://github.com/nextflow-io/nextflow/commit/4a165f43b0b647cdb0c3293adfeac0db7bc09cab)

Upgrade sphinx-rtd-theme to 0.5.2 worked fine with me. Thanks.

```
pip install -U sphinx-rtd-theme
```
```
Installing collected packages: sphinx-rtd-theme
  Attempting uninstall: sphinx-rtd-theme
    Found existing installation: sphinx-rtd-theme 0.5.0
    Uninstalling sphinx-rtd-theme-0.5.0:
      Successfully uninstalled sphinx-rtd-theme-0.5.0
Successfully installed sphinx-rtd-theme-0.5.2
```
